StardewArchipelago/Extensions/GameLocationExtensions.cs

```diff
@@ -305,6 +316,18 @@ private static IEnumerable<Point> GetDoorWarpPoints(GameLocation origin, string
             }
         }
 
+        private static IEnumerable<Point> GetBuildingWarpPoints(GameLocation origin, string destinationName)

```

Comment:
The destination is not considered at all by this method, so it will return wrong warp points if there are many candidates

Reply:
destinationName is considered against the **generic** indoor name now. e.g. "Shed". If multiple sheds, all will be considered "Shed".

NameOrUniqueName can be used to get a specific instance, however, it may break some matching with how the entrance randomizer creates the links at this moment. e.g. "Farm_To_Shed" would not match on "Farm_To_Shedf3e2e403-5834-4aa4-b818-51da90e7ca08".